packages/manifest-helper/src/utils/error.ts

```diff
@@ -0,0 +1,5 @@
+export const getErrorMessage = (err: unknown) => {
+  if (err instanceof Error) return err.message
```

Comment:
TS consider all exception caught as of the type `unknown` and for this reason, if I try to call the property `message` of the caught error instance, TS is going to complain about not recognizing `message` as a valid property.

```
try {
...
} catch (err) {
    console.log(err.message)
    //TS will complain here
}
```

![image](https://github.com/storyblok/field-plugin/assets/1240591/07ef8299-40d7-4c8b-a8c9-6daa2b768df3)

So, based on [this tutorial](https://kentcdodds.com/blog/get-a-catch-block-error-message-with-typescript) I created this 'helper function' to deal better with this scenario.

However, If there is any better way to deal with it, I'd be glad to know, because I, myself, didn't find this approach so beautiful. 😆

Reply:
As far as I know this is a standard approach to handle catch clauses so its ok this way.